### PR TITLE
[EJBCLIENT-311] Add timeout for discovering the destination of an EJB request

### PR DESCRIPTION
**Pull request by iweiss**
Issue: https://issues.jboss.org/browse/EJBCLIENT-311

Don't merge: Depends on https://issues.jboss.org/browse/WFDISC-34

### CODE REVIEW

File: src/test/resources/no-protocol-jboss-ejb-client.properties [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#
+# JBoss, Home of Professional Open Source.
+# Copyright 2010 Red Hat, Inc., and individual contributors
```
Comment (tadamski) [inline src/test/resources/no-protocol-jboss-ejb-client.properties]: typo (2019)

File: src/test/java/org/jboss/ejb/client/test/RemoteDiscoveryTimeoutTestCase.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
+package org.jboss.ejb.client.test;
```
Comment (tadamski) [inline src/test/java/org/jboss/ejb/client/test/RemoteDiscoveryTimeoutTestCase.java]: Please add the license header

**Comment by iweiss** [comment]
@tadamski should be all right now

File: src/test/java/org/jboss/ejb/client/test/RemoteDiscoveryTimeoutTestCase.java (hunk 2 above)
Comment (tadamski) [reply src/test/java/org/jboss/ejb/client/test/RemoteDiscoveryTimeoutTestCase.java]: Hmm, I can't see anything :/ Did you pushed that?

**Pull request merged**